src/components/content/catalog/services/UnregisterService.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import React from 'react';
+import { Button, message, Popconfirm } from 'antd';
+import { serviceVendorApi } from '../../../../xpanse-api/xpanseRestApiClient';
+
+function UnregisterService({ id }: { id: string }): JSX.Element {
+    function deleteService() {
+        serviceVendorApi
+            .unregister(id)
+            .then((rsp) => {
+                return rsp.success;
+            })
+            .catch(() => {
+                return false;
+            });
+        return true;
+    }
+
+    const sleep = (ms: number) => new Promise((r) => setTimeout(r, ms));
+
+    const confirm = async () => {
+        if (deleteService()) {
+            message.success('Delete successfully.', 0.5);
```

Comment:
Would be good to have a alert message like all other pages instead of just a pop message. We need to maintain this uniformity on all pages.

Reply:
Agree, and also it would be better with `Unregistered successfully`.